src/document/__tests__/DocumentAnnotator-test.ts

```diff
@@ -286,15 +287,27 @@ describe('DocumentAnnotator', () => {
         });
 
         test('should add and remove is highlighting class if mode changes', () => {
-            expect(annotator.annotatedEl?.classList.contains('ba-is-highlighting')).toBe(false);
+            expect(
+                annotator.annotatedEl?.classList.contains(
+                    ANNOTATION_CLASSES[Mode.HIGHLIGHT] || 'ba-is-create--higlight',
```

Comment:
Could we just use the string `ba-is-create--highlight` instead for these expectations?

Reply:
I originally tried to use `ANNOTATION_CLASSES[Mode.HIGHLIGHT]` but then there was an error due to the contains method not accepting a potential param of `undefined` so I added the || but now it looks like it'll be better if we just define the parameter as `ba-is-create--highlight` instead.